admin_guide/high_availability.adoc

```diff
@@ -580,6 +588,54 @@ the list of VIPs.  The vrrp-ids may have values in the range 1..255.
 
 When there are multiple ipfailover deployment configuration care must be taken to specify `--vrrp-id-offset` so that there is room to increase the number of VIPS in the deployment configuration and none of the vrrp-id ranges overlap.
```

Comment:
That should be VIPs.

Did we ever define VIP as Virtual IP?

Reply:
It did in the overview, but for the sake of topic reusability, I've added this in a few times. Good eye.